src/forms/TopbarSearchForm/TopbarSearchForm.js

```diff
@@ -30,33 +29,57 @@ class TopbarSearchFormComponent extends Component {
   }
 
   render() {
-    const { rootClassName, className, desktopInputRoot, intl, isMobile } = this.props;
+    return (
+      <FinalForm
+        {...this.props}
+        render={({ rootClassName, className, desktopInputRoot, intl, isMobile }) => {
+          const classes = classNames(rootClassName, className);
+          const desktopInputRootClass = desktopInputRoot || css.desktopInputRoot;
 
-    const classes = classNames(rootClassName, className);
-    const desktopInputRootClass = desktopInputRoot || css.desktopInputRoot;
+          // Allow form submit only when the place has changed
+          const preventFormSubmit = e => e.preventDefault();
 
-    // Allow form submit only when the place has changed
-    const preventFormSubmit = e => e.preventDefault();
+          return (
+            <Form className={classes} onSubmit={preventFormSubmit}>
+              <Field
+                name="location"
+                format={null}
+                render={({ input, meta }) => {
```

Comment:
We haven't used this spread format of props with other components. It helps to debug code if there's full prop list available.